addons/web_editor/static/src/components/media_dialog/media_dialog.js

```diff
@@ -67,6 +67,10 @@ export class MediaDialog extends Component {
         return this.tabs[0].id;
     }
 
+    get activeTab() {
```

Comment:
As discussued in https://github.com/odoo/odoo/pull/100253, this commit should be removed right?

Reply:
Yes